code/modules/mob/living/carbon/human/species_types/vox.dm

```diff
@@ -0,0 +1,176 @@
+/datum/species/vox
+	name = "Vox"
+	plural_form = "Vox"
+	id = SPECIES_VOX
+	is_dimorphic = FALSE
+	generate_husk_icon = TRUE
+	species_traits = list(EYECOLOR, HAS_TAIL, HAS_FLESH, HAS_BONE, HAIRCOLOR, FACEHAIRCOLOR, MUTCOLORS, MUTCOLORS_SECONDARY) // Robust, but cannot be cloned easily.
+	inherent_traits = list(TRAIT_RESISTCOLD, TRAIT_NOCLONE)
+	mutant_bodyparts = list("vox_quills", "vox_body_markings", "vox_facial_quills", "vox_tail", "vox_tail_markings")
+	default_features = list("vox_quills" = "None", "vox_facial_quills" = "None", "vox_body_markings" = "None", "vox_tail" = "lime", "vox_tail_markings" = "None", "vox_skin_tone" = "lime")
+	attack_verbs = list("scratch", "claw")
+	attack_effect = ATTACK_EFFECT_CLAW
+	attack_sound = 'sound/weapons/slash.ogg'
+	miss_sound = 'sound/weapons/slashmiss.ogg'
+	screamsound = list('sound/voice/vox/shriek1.ogg'/*, 'sound/voice/vox/ashriek.ogg'*/)
+	cough_sound = 'sound/voice/vox/shriekcough.ogg'
+	sneeze_sound = 'sound/voice/vox/shrieksneeze.ogg'
+	mutantbrain = /obj/item/organ/brain/vox // Brain damage on EMP
+	mutantheart = /obj/item/organ/heart/vox
+	mutantliver = /obj/item/organ/liver/vox // Liver damage on EMP
+	mutantstomach = /obj/item/organ/stomach/vox // Disgust on EMP
+	mutanttongue = /obj/item/organ/tongue/vox
+	mutantlungs = /obj/item/organ/lungs/vox // Causes them to.. gasp.
+	mutantears = /obj/item/organ/ears/vox // Very brief deafness
+	mutanteyes = /obj/item/organ/eyes/vox // Quick hallucination
+	mutanttail = /obj/item/organ/tail/vox
+	mutantappendix = /obj/item/organ/appendix/vox
+	meat = /obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/vox
```

Comment:
/obj/item/reagent_containers/food/snacks/meat/slab/chicken

Reply:
I thought about this but figured it would be better to keep it as a subtype of human meat (like the other alien species) for consistency sake and to maintain compatibility with all the stuff that uses its subtypes, like meaty meteors

Reply:
chiken